app/components/client_upgrade_listener/client_upgrade_listener.js

```diff
@@ -110,27 +140,39 @@ class ClientUpgradeListener extends PureComponent {
         });
 
         this.toggleUpgradeMessage(false);
-    }
+    };
 
     handleLearnMore = () => {
-        this.props.navigator.dismissAllModals({animationType: 'none'});
+        const {intl} = this.context;
+        this.props.navigator.dismissModal({animationType: 'none'});
```

Comment:
Did you mean to change this from dismissAllModals to dismissModal?

Reply:
Yes